src/main/groovy/io/github/cosmicsilence/scalafix/ScalafixPlugin.groovy

```diff
@@ -25,17 +25,18 @@ class ScalafixPlugin implements Plugin<Project> {
 
     @Override
     void apply(Project project) {
-        if (!project.plugins.hasPlugin(ScalaPlugin)) {
-            throw new GradleException("'scala' plugin must be applied before 'scalafix'")
-        }
 
         def extension = project.extensions.create(EXTENSION, ScalafixExtension, project)
         def customRulesConfiguration = project.configurations.create(CUSTOM_RULES_CONFIGURATION)
         customRulesConfiguration.description = "Dependencies containing custom Scalafix rules"
 
-        configureTasks(project, extension)
-
         project.afterEvaluate {
+            if (!project.plugins.hasPlugin(ScalaPlugin)) {
```

Comment:
are we including this check so that the order the plugins are applied does not affect? In that case...could we ask a quick test to prove the order doesn't matter? :D

Reply:
this check was already there. I'm moving it into the `afterEvaluate` as well as now the order both `scala` and `scalafix` plugins are applied is no longer important. I will add the test now 👍